### PR TITLE
Add city and country to tracking data

### PR DESCRIPTION
**Pull request by mattandrews**
@jennysivapalan does this look right to you?

Should add town and country to the tracking when new members sign up (nothing more specific than that!) – so we can track signups by location.

### CODE REVIEW

File: frontend/app/services/MemberService.scala [hunk 1]
```diff
@@ -178,7 +178,9 @@ trait MemberService extends LazyLogging with ActivityTracking {
             deliveryPostcode = form.map(_.deliveryAddress.postCode),
             billingPostcode = form.flatMap(f => f.billingAddress.map(_.postCode)).orElse(form.map(_.deliveryAddress.postCode)),
             subscriptionPaymentAnnual = Some(annual),
-            marketingChoices = None
+            marketingChoices = None,
+            city = form.map(_.deliveryAddress.town),
+            country = form.map(_.deliveryAddress.country.name)
```
Comment (jennysivapalan) [inline frontend/app/services/MemberService.scala]: Looks like you have added this to `membershipUpgrade` (which might be useful?) but you need to add this to where we track `membershipRegistration`

Comment (jennysivapalan) [inline frontend/app/services/MemberService.scala]: Ignore me...

File: frontend/app/tracking/ActivityTracking.scala [hunk 2]
```diff
@@ -75,6 +77,8 @@ case class MemberData(salesforceContactId: String,
         deliveryPostcode.map("deliveryPostcode" -> truncatePostcode(_)) ++
         billingPostcode.map("billingPostcode" -> truncatePostcode(_)) ++
         subscriptionPlan.map("subscriptionPlan" -> _) ++
+        city.map("city" -> _) ++
+        country.map("country" -> _) ++
```
Comment (mattandrews) [inline frontend/app/tracking/ActivityTracking.scala]: this right @jennysivapalan?

Comment (jennysivapalan) [inline frontend/app/tracking/ActivityTracking.scala]: :+1:

**Comment by jennysivapalan** [comment]
:+1: :tokyo_tower:

**Comment by mattandrews** [comment]
@jamesoram, is this okay with you?

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**